rfcs/react-components/components/combobox-spacebar.md

```diff
@@ -0,0 +1,115 @@
+# RFC: Combobox spacebar behavior
+
+@smhigley @jurokapsiar
+
+## Summary
+
+This RFC deals with how to provide space-to-type behavior in v9 Comboboxes.
+
+## Background
+
+The central issue is that spacebar is both a common selection key and a typeable character, and Combobox is a control with both selection and typing functionality.
+
+While this RFC deals only with Combobox and not Dropdown, it is important to note that semantically Combobox and Dropdown are essentially the same. Expectations among screen reader users for keyboard behavior seem to easily carry over between the two. Non-editable comboboxes are a more common and simple control, and are likely influencing keyboard expectations with editable comboboxes.
+
+Another important factor to note is that using Enter to select is not as common as might be expected, since it is also the key that submits a form, and often avoided within forms for that reason. This is less of an issue with a standalone combobox where selection and submission are not different interactions.
+
+Currently, there is an agreement on inserting a space character when freeform variant is used and the user has not interacted with the dropdown list. See [feat: react-combobox space conditionally inserts character when freeform is true](https://github.com/microsoft/fluentui/pull/27025) for details.
+
+## Precedence
```

Comment:
I would also note that the APG guidance doesn't seem to specify that spacebar should be used to select https://www.w3.org/WAI/ARIA/apg/patterns/combobox/